src/os/bluestore/bluestore_tool.cc

```diff
@@ -173,7 +173,8 @@ BlueFS *open_bluefs(
   const vector<string>& devs)
 {
   validate_path(cct, path, true);
-  BlueFS *fs = new BlueFS(cct);
+  uint64_t alloc_size = 0;
```

Comment:
nit: leave a comment to future generations  that we don't need alloc_size here? ;)